content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -56,76 +67,94 @@ turn, notify the customer that the order (or a part of it) is on its way.
+
+Additionally, as Amazon makes a single monthly payment, selecting all the invoices linked to a
+single payment is necessary when registering payments.
+
+To do that, used the appropriate :guilabel:`Journal` dedicated to Amazon payments, and select
+:guilabel:`Batch Deposit` as the :guilabel:`Payment Method`.
 
-In addition, as Amazon makes a single monthly payment, selecting all the invoices linked to a single
-payment is necessary when registering payments. Use the dedicated `Amazon payments`
-:guilabel:`Journal` and select :guilabel:`Batch Deposit` as the :guilabel:`Payment Method`. Then,
-select all the payments generated and click :menuselection:`Actions --> Create batch payment -->
-Validate`.
+Then, select all the generated payments, and click :menuselection:`Actions --> Create batch payment
+--> Validate`.
 
 .. tip::
-   The same can be done with vendor bills from Amazon dedicated to commissions. When the balance is
-   received in the bank account at the end of the month and the banks statements are recorded,
-   credit the Amazon intermediary account by the amount received.
+   This same action can be performed with vendor bills from Amazon dedicated to commissions.
 
-Follow your Amazon sales in sales reporting
-===========================================
+   When the balance is received in the bank account at the end of the month, and the bank statements
+   are recorded, credit the Amazon intermediary account by the amount received.
 
-As a sales team is set on your account under the tab **Order Follow-up**, this helps you give quick
-glances at the figures in just a few clicks in Sales reporting. By default, your account's sales
-team is shared between all of your company's accounts.
+Follow Amazon sales in sales reporting
+======================================
 
-If you wish, you can change the sales team on your account for another to perform a separate
+On the Amazon account, a sales team is set under the :guilabel:`Order Follow-up` tab.
+
+This gives quick access to important metrics related to Sales reporting. By default, the account's
+sales team is shared between all the company's accounts.
+
+If desired, the sales team on the account can be changed for another, in order to perform a separate
 reporting for the sales of this account.
 
 .. tip::
-   It is also possible to perform reporting on a per-marketplace basis in a similar fashion. First,
-   remove the marketplace you wish to track separately from the list of synchronized marketplaces.
-   Then, create another registration for this account and remove all marketplaces, except the one to
-   isolate from the others. Finally, assign another sales team to one of the two registrations of
-   your account.
+   It is also possible to perform reporting on a per-marketplace basis in a similar fashion.
+
+   First, remove the desired marketplace from the list of synchronized marketplaces.
```

Comment:
Again, can we include where to find the list of synchronized marketplaces in Odoo?